translations/README.zh-cn.md

```diff
@@ -117,10 +117,11 @@ npm run convert
 
 ## 其它课程
 
-我们的组生成了其它课程！查一下：
+我们的团队制作了其它课程！查一下：
 
-- [网站开发（Web Dev for Beginners）](https://aka.ms/webdev-beginners)
+- [Web 开发（Web Dev for Beginners）](https://aka.ms/webdev-beginners)
```

Comment:
@ChungZH Do you think we should leave it as `网站开发` since we have already provided the English translation `Web Dev` beside it, or is `Web 开发` significantly more common?